api/controllers/FlairController.js

```diff
@@ -145,7 +145,7 @@ module.exports = {
       if (events.length) {
         var then = moment(events[0].createdAt);
         then.add(2, 'minutes');
-        if (then.isAfter(now) && false) {
```

Comment:
I've also had to do that in the past. We could change it to `if (sails.config.environment === 'production' && otherCondition)` or something, though it's not a huge issue either way.